libs/coin-modules/coin-stellar/src/api/index.integ.test.ts

```diff
@@ -91,5 +94,40 @@ describe.skip("Stellar Api", () => {
 
       expect(result.length).toEqual(188);
     });
+
+    it("should use estimated fees when user does not provide them for crafting a transaction", async () => {
+      const result = await module.craftTransaction({
+        type: "send",
+        sender: address,
+        recipient: "GD6QELUZPSKPRWVXOQ3F6GBF4OBRMCHO5PHREXH4ZRTPJAG7V5MD7JGX",
+        amount: BigInt(1_000_000),
+      });
+
+      expect(result.length).toEqual(188);
+      expect(logicCraftTransactionSpy).toHaveBeenCalledWith(
+        expect.any(Object),
+        expect.objectContaining({ fee: 100n }),
```

Comment:
This value could change, don't we want to verify it's the output of estimate fees ? => moving both tests to unit tests